src/main/java/uk/gov/hmcts/reform/et/syaapi/notification/NotificationsProperties.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.et.syaapi.notification;
+
+import lombok.Data;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.validation.annotation.Validated;
+
+/**
+ * Holds gov-notify api key and templateId details.
+ */
+
+@Validated
+@Data
+public class NotificationsProperties {
+
+    @Value("${notifications.govNotifyApiKey}")
+    private String govNotifyApiKey;
+
+    @Value("${notifications.sampleEmailTemplateId}")
```

Comment:
Does this need to be extendable for future use? maybe need some form of lookup for different templates